### DISCUSSION TITLE
Allow LNURL-Auth primary auth handler

### DISCUSSION THREAD

**Discussion by status21** [btcpayserver/btcpayserver#5092] [Ideas / Feature Requests]
Would it be possible to integrate log in with lightning using Zeus or other wallets instead of username and password?

**Comment by Kukks** [comment]
We currently allow that as a two factor authentication. We did want to
allow fido2 or lnurl auth as primary auth handlers though.

Concept ack

**Comment by pavlenex** [comment]
@Kukks we abandon this idea for now, right?
